Art/Groot-2001/style.css

```diff
@@ -1,42 +1,56 @@
-body {
-  background: radial-gradient(yellow, rgb(255 231 0 / 57%), white);
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
   display: flex;
   justify-content: center;
   align-items: center;
   height: 100vh;
+  background-color: whitesmoke;
+}
+
+#B {
+  animation: rotate 5s infinite linear;
+}
+
+#M {
+  animation: bounce 3s infinite ease-in-out;
 }
 
-.loader {
-  width: 100px;
-  height: 100px;
-  background: linear-gradient(
-    165deg,
-    #ffc800 0%,
-    rgb(255 127 0 / 83%) 40%,
-    #ff000059 98%,
-    rgb(10, 10, 10) 100%
-  );
-  border-radius: 50%;
-  position: relative;
-}
-
-.loader:before {
-  position: absolute;
-  content: '';
-  width: 100%;
-  height: 100%;
-  border-radius: 100%;
-  border-bottom: 0 solid rgb(255 127 0);
-  box-shadow: 0 -10px 20px 20px #ffffff40 inset,
-    0 -5px 15px 10px #ffffff50 inset, 0 -2px 5px #ffffff80 inset,
-    0 -3px 2px #ffffffbb inset, 0 2px 0px #ffffff, 0 2px 3px #ffffff,
-    0 5px 5px #ffffff90, 0 10px 15px #ffffff60, 1px 0px 20px 20px #ffc800;
-  filter: blur(3px);
-  animation: 2s rotate linear infinite;
+#H {
+  animation: flip 4s infinite ease-in-out;
 }
 
 @keyframes rotate {
   100% {
     transform: rotate(360deg);
   }
 }
+
+@keyframes bounce {
+  0%,
+  100% {
+    transform: translateY(0);
+  }
+
+  50% {
+    transform: translateY(-30px);
+  }
+}
+
+@keyframes flip {
+  0% {
+    transform: scaleY(1);
+  }
+
+  50% {
+    transform: scaleY(-1);
+  }
+
+  100% {
+    transform: scaleY(1);
+  }
+}
```

Comment:
# Feedback - unused file ?
If you do not use this css file please remove it or add the missing files attached to that development.

Then, this will be reviewed to check if it is good to go :)